nbdev/export2html.py

```diff
@@ -553,7 +553,11 @@ def notebook2html(fname=None, force_all=False, n_workers=None, cls=HTMLExporter,
                 files.append(fname)
     if len(files)==0: print("No notebooks were modified")
     else:
-        passed = parallel(_notebook2html, files, n_workers=n_workers, cls=cls,
+        if sys.platform == "win32":
```

Comment:
I believe you should be able to just set `n_workers=0` for win32. fastcore's `parallel` disables multiprocessing in that case.

Reply:
@jph00 , Thank your reply. I'm afraid that we could NOT use n_workers=0 here since it still spawns a new process even n_wordkers=0

we could verify it by the below snippt.
```
from concurrent.futures import ProcessPoolExecutor as Executor

def func():
        print(__name__)
        print('Hello')

if __name__ == '__main__':
        print(__name__)
        with Executor(1) as executor:
                future=executor.submit(func)
                print(future.result())
```

we can see the result is
__main__
__mp_main__
Hello
None

ref:
https://stackoverflow.com/questions/61860800/running-a-processpoolexecutor-in-ipython

Reply:
@jph00 , Thank your reply. it was stuck when if n_workers=0. 
I find it's caused by https://github.com/fastai/fastcore/blob/8ddf4dde118dea53daf849d7a5f2dc425ca49390/fastcore/parallel.py#L84-L85.
The Manager().Lock() spawns a new process.

Could we don't set [self.lock = Manager().Lock()](url) when it is not_parallel?